frontend/stories/inputs/select/FormSelect.stories.tsx

```diff
@@ -15,58 +15,96 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import { FormSelect } from '@controls/formSelect/formSelect.component';
-import { MenuItem } from '@mui/material';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { useForm } from 'react-hook-form';
+import ClearIcon from '@assets/icons/controls/clear_circle.svg';
+import ChevronIcon from '@assets/icons/chevron.svg';
+import CalendarIcon from '@assets/icons/outlined/calendar-outlined.svg';
+import PrintIcon from '@assets/icons/print.svg';
+import { SelectMenuItem } from '@controls/formSelect/formSelectMenutItem.styles';
 import { FormContainer } from '../FormInput.styles';
+import { IconContainer, ItemContainer } from './FormSelect.styles';
 
 export default {
-	title: 'Inputs/Select/FormSelect',
+	title: 'Inputs/Select/FormSingleSelect',
 	argTypes: {
 		label: {
 			type: 'string',
 		},
 		defaultValue: {
-			type: 'string',
+			type: 'number',
+		},
+		disabled: {
+			type: 'boolean',
 		},
-		values: {
-			control: 'array',
+		search: {
+			type: 'boolean',
 		},
 		formError: {
 			type: 'string',
 		},
-		disabled: {
-			type: 'boolean',
-		},
 	},
 	component: FormSelect,
-	parameters: { controls: { exclude: ['control', 'ref'] } },
+	parameters: { controls: { exclude: ['control', 'ref', 'onItemClick', 'itemIsSelected', 'selectedOptionsTooltip', 'renderValueTooltip'] } },
 } as ComponentMeta<typeof FormSelect>;
 
-const Controlled: ComponentStory<typeof FormSelect> = ({ formError, values, ...args }: any) => {
+const Controlled: ComponentStory<typeof FormSelect> = ({ formError, ...args }: any) => {
 	const { control } = useForm({ mode: 'onChange' });
 
 	return (
 		<FormContainer>
```

Comment:
I'm not sure how important this is, so feel free to ignore. But I've noticed you can't use keyboard controls to navigate/select values in this select (whereas you can with select base)

Reply:
I know, unfortunately that feature got lost with the introduction of the custom scrollbar. MUI does not support a wrapper around the MenuItems (which it wants to appear as direct children). All the logic that you see in the new FormSelect is almost only to "adapt" the custom scrollbar to the FormSelect, but that functionality that you mentioned was lost this way. Probably the custom scrollbar was not a great Idea